src/Resources/External/CreditNoteResource.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
```

Comment:
using outside. wonder if we can stop resharper from putting them here by default 

Or perhaps just start using the new namespace style

Reply:
it is certainly  a pain on every PR to have to double checking all the usings